### PR TITLE
update: footer and theme and text

### PR DESCRIPTION
**Pull request by tamascsaba**

### CODE REVIEW

File: 2023/src/theme.ts [hunk 1]
```diff
@@ -27,15 +27,36 @@ export const theme: DefaultTheme = {
...
+      .map(n => `"${n}"`)
+      .join(", "),
   },
   fontSizes: {
     hero: "6rem",
     title: "4rem",
     subTitle: "2.6rem",
   },
   breakpoints: {
-    mobile: "@media (max-width: 576px)",
```
Comment (martinheidegger) [inline 2023/src/theme.ts:38]: The breakpoints are a separate PR I think.

Comment (tamascsaba) [reply 2023/src/theme.ts:38]: You are right, just a sec and I correct it.

Comment (tamascsaba) [reply 2023/src/theme.ts]: fixed :)

File: 2023/src/components/Footer.tsx [hunk 2]
```diff
@@ -60,22 +69,32 @@ export function Footer(_props: Props) {
...
+        <li>
+          <ExternalLink href="https://nodejs.jp/" target="_blank">
+            Japan Node.js Association
+          </ExternalLink>
+        </li>
+        <li>
+          <Link to="/code-of-conduct/">{t("code-of-conduct")}</Link>
+        </li>
+        <li>
+          <Link to="/specified-commercial-transactions-act/">
```
Comment (martinheidegger) [inline 2023/src/components/Footer.tsx:94]: The link here changed.

File: 2023/src/theme.ts (hunk 1 above)
Comment (martinheidegger) [reply 2023/src/theme.ts:38]: I don't think its fixed, as it is still part of this pr :sweat_smile:

Comment (tamascsaba) [reply 2023/src/theme.ts]: Sorry, I forgot to remove stashed changes 😢

**Comment by martinheidegger** [comment]
Screen Shot 2023-11-16 at 00 46 51

At intermediate sizes the layout is a bit odd.

**Comment by tamascsaba** [comment]
I changed the style a little, how do you like it?: image

**Review by martinheidegger — APPROVED** [review]
LGTM

**Review by Leko — APPROVED** [review]
LGTM!

**Pull request merged**